src/Symfony/Component/Notifier/Bridge/GoogleChat/GoogleChatOptions.php

```diff
@@ -71,6 +73,16 @@ public function card(array $card): static
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function cardV2(array $card): static
+    {
+        $this->options['cardsV2'][] = $card;
```

Comment:
please use this `cardsV2` key in the `card()` method and remove the `cardV2` method

Reply:
I don't think so, in google developer api doc, we can still use card and cardV..
The old cards are not compatible with the v2 so it will break old code.

I will create a new PR to help create cards using a factory instead of juste injecting the all cards array.

Reply:
We should only support the Not deprecated version V2

Reply:
@OskarStark  OK 
DONE

Reply:
supporting only the v2 is a BC break as currently the method supports the v1 format for the argument. We cannot do that in a minor version.

Reply:
So you propose to add a second method `cardV2()` like initially done? I mean, it's not about our BC promise, but the 2rd party API changes. I am not sure how our BC promise can/should handle such situation

Reply:
I would indeed be on the safe side and add a cardv2 method instead.

Reply:
Ok let's go ahead than and deprecate the card() method 👍🏻

Sorry for the confusion

Reply:
Done ✅